airbyte-integrations/connectors/source-s3/source_s3/v4/source.py

```diff
@@ -21,14 +21,17 @@
 
 
 class SourceS3(FileBasedSource):
-    def read_config(self, config_path: str) -> Mapping[str, Any]:
+    _concurrency_level = DEFAULT_CONCURRENCY
```

Comment:
is there value in allowing the user to configure the concurrency level?

Reply:
This is to allow each source to turn on concurrency intentionally, instead of accidentally deploying it when the next version of the connector is deployed.